numpyro/mcmc.py

```diff
@@ -32,7 +32,20 @@ def _sample_momentum(unpack_fn, inverse_mass_matrix, rng):
         raise NotImplementedError
 
 
-def hmc_kernel(potential_fn, kinetic_fn, algo='NUTS'):
+def _euclidean_ke(r, inverse_mass_matrix):
```

Comment:
I was wondering if we should reverse the order of arguments. That way, we can apply partial with `inverse_mass_matrix` if we would like to in the future, and with the signature of both `potential_energy` and `kinetic_energy` can be the same.